rest-api-spec/src/main/resources/rest-api-spec/api/snapshot.repository_analyze.json

```
@@ -36,6 +36,10 @@
         "type":"number",
         "description":"Number of operations to run concurrently during the test. Defaults to 10."
       },
+      "register_operation_count":{
+        "type":"number",
+        "description":"The minimum number of linearizable register operations to perform in total. For realistic experiments, you should set it to at least `100`. Defaults to 10."
```

Nit: I think we can simplify it by removing the sentence about "realistic experments", i.e.
```suggestion
        "description":"The minimum number of linearizable register operations to perform in total. Defaults to 10."
```
This is more consistent with others which do not touch "realistic experiments" either.

---

Thanks! The docs do mention "realistic experiments" and we're merging the docs and API specification. I should probably have updated the others. The rest-api-spec is only used for the YAML tests and the PHP client, all others clients use https://github.com/elastic/elasticsearch-specification.

---

Yeah however consistency would be great. Thanks!